src/components/Banner.tsx

```diff
@@ -1,9 +1,9 @@
 import Button from './button/Button';
 import { PageTitle, Title, Text } from "./text/Text";
 
-import ethereum from "../images/ethereum.png";
-import arbitrum from "../images/arbitrum.png";
-import uniswap from "../images/uniswap.png";
+import ethereum from "../images/ethereum.svg";
```

Comment:
I believe you were using a library to shorten the import url ? something like '@src/xyz/components...'